ratis-client/src/main/java/org/apache/ratis/client/api/BlockingApi.java

```diff
@@ -45,14 +45,19 @@ default RaftClientReply sendReadOnly(Message message) throws IOException {
     return sendReadOnly(message, null);
   }
 
+  default RaftClientReply sendReadOnly(Message message, RaftPeerId server) throws IOException {
+    return sendReadOnly(message, server, false);
+  }
+
   /**
    * Send the given readonly message to the raft service.
    *
    * @param message The request message.
    * @param server The target server.  When server == null, send the message to the leader.
+   * @param readIndex weather use linearizable read.
    * @return the reply.
    */
-  RaftClientReply sendReadOnly(Message message, RaftPeerId server) throws IOException;
+  RaftClientReply sendReadOnly(Message message, RaftPeerId server, boolean readIndex) throws IOException;
```

Comment:
Shall we use `Read.Option` as the third parameter? We may introduce lease-based linearizable read in the future, and boolean won't work then.

Reply:
just add a new interface, PTAL, thanks!